plugins/org.komodo.relational/src/org/komodo/relational/workspace/Workspace.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * See the COPYRIGHT.txt file distributed with this work for information
+ * regarding copyright ownership.  Some portions may be licensed
+ * to Red Hat, Inc. under one or more contributor license agreements.
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * 
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301 USA.
+ */
+package org.komodo.relational.workspace;
+
+import org.komodo.relational.model.Model;
+import org.komodo.relational.model.Schema;
+import org.komodo.spi.KException;
+import org.komodo.spi.repository.Repository.UnitOfWork;
+
+/**
+ *
+ */
+public interface Workspace {
```

Comment:
I would vote to remove and refactor into `WorkspaceManager`. I don't think the workspace is a relational object. I believe it is more like our `Repository` object.